jujucrashdump/crashdump.py

```diff
@@ -178,12 +192,29 @@ def __init__(self, model, max_size, extra_dirs, output_dir=None,
         self.timeout = timeout
         self.journalctl = journalctl
 
+    def get_all(self):
+        machines = {}
+        juju_status = yaml.load(open('juju_status.yaml', 'r'),
+                                Loader=yaml.FullLoader)
+        for machine, machine_data in juju_status['machines'].iteritems():
```

Comment:
Just use .items() - it's fine in both python2 and python3 that way.

Reply:
good point